src/common/updating/toast_notifications_helper.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+namespace updating
+{
+    struct new_version_download_info;
+}
+
+namespace notifications

```

Comment:
the same namespace is used for toast notifications API library, let's move it inside `updating`. otherwise some context is lost. e.g. compare:
`notifications::show_unavailable()` vs `updating::notifications::show_unavailable`